### ISSUE TITLE
template: error 500 page should have nomad template

### ISSUE THREAD

**Issue by engerm** [RagtagOpen/nomad#247]
<img width="1504" alt="screen shot 2017-10-07 at 8 35 35 pm" src="https://user-images.githubusercontent.com/17053096/31312906-4bc79bb8-ab9f-11e7-8e8f-7bff7fbbe8a4.png">

Sample: modify the leaving_from_lat value when creating a new carpool
```
POST /carpools/new HTTP/1.1
Host: nomad.ragtag.org
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:55.0) Gecko/20100101 Firefox/55.0
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate
Content-Type: application/x-www-form-urlencoded
Content-Length: 588
Referer: https://nomad.ragtag.org/carpools/new
Cookie: session=.eJwdj0FuQyEMRK9Ssc7CGGMgl4nAHqtVpVb6P1lFuXto9_PmzTzTLQ6cn-l6Px64pNuXp2siAHlGjBrBNDxz7wpxj8zcnTuNPOoQsE7y4lbmTnVqXIxRCkSYfVSbk5rmEKpF-8rSbZHX2VomJWkROYhXYKENNkOfGzNJl2TnEbf77zd-9h4bmLui-qZb0zFoO7jUKtTrAi3nuaC6uceJ4__EM308_q4U5YYm28OStz8yAkZY7gq19Hq9AZvWSdA.DLsDuw.I3u7k8WCq6Vax7pVCtGIW2eFg1M; remember_token=03627e74-f022-418c-bf1e-fec0ebdd6e6c|98172265deaf71f9415c394346b4ecf6018d84a6c6a9da8afde8da7148aa9a9952c74484f43bdb47e5b2850fcd68048c1c924a2bef4fd9888770f0ab743a6583
DNT: 1
Connection: close
Upgrade-Insecure-Requests: 1

leaving_from_lat=38.9027825999999'`9&leaving_from_lon=-77.06171999999998&going_to_id=3&going_to_lat=38.9317319&going_to_lon=-77.03816699999999&csrf_token=ImM5ZWEwNTM1ZGIwZDc3Njk5MDMzZTIzNTU0MDg1YmUwYmQyYWJlNjYi.DLsEPg.uYOVAC-6LmOPGR-kdr36uQGEyj0&leaving_from=3113+K+St+NW%2C+Washington%2C+DC%2C+United+States&depart_time=11%2F01%2F2017+20%3A30&leaving_from_lat=&leaving_from_lon=&going_to_list=3&going_to_text=3221+Mt+Pleasant+St+NW%2C+Washington+D.C.%2C+DC+20010%2C+United+States&going_to_id=&return_time=11%2F02%2F2017+20%3A30&going_to_lat=&going_to_lon=&car_size=8&submit=Add+Your+Ride
```

**Comment by iandees** [comment]
The 500 page uses a template now.

**Comment by iandees** [comment]
Fixed in #56.
